crates/m3/tests/merkle_tree.rs

```diff
@@ -807,8 +831,11 @@ mod arithmetization {
 
 			let parent_depth = table.add_committed("parent_depth");
 
-			// TODO: constrain child depth to be parent depth + 1 with incr lookup.
-			let child_depth = table.add_committed("child_depth");
+			let one = table.add_constant("one", [B1::ONE]);
```

Comment:
You shouldn't need a constant column, you can flush with an `OracleOrConst::Const`